index.html

```diff
@@ -0,0 +1,183 @@
+          <div class="card">
+            <img src="./images/home/research.svg" alt="icon" />
+            <h4 class="card-title">Lecture</h4>
+            <p class="card-desc">
+              listen to the speakers from various countries about the message of
+              sharing and opening.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/design.svg" alt="icon" />
+            <h4 class="card-title">CC Exhibition</h4>
+            <p class="card-desc">
+              Appreciate various creations applying CC licence of
+              artists,organized from Art center Nabi.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/networking.svg" alt="network-icon" />
+            <h4 class="card-title">Forum</h4>
+            <p class="card-desc">
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/discussion.svg" alt="icon" />
+            <h4 class="card-title">Workshop</h4>
+            <p class="card-desc">
+              Try creating your own work using open source licence rather than
+              just watching it.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/knowledge.svg" alt="icon" />
+            <h4 class="card-title">CC Ignite</h4>
+            <p class="card-desc">
+              Get opportuntities to network with CC affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </article>
+        <button  class="main-btn">
+          <p class="join-text">Join CC Global Summit 2015 </p>
+        </button>
+        <a class="more" href="#" id="active-link">See the whole programe</a>
+      </section>
+      <section class="feature-section">
+        <h3 class="sec-title">Feature speakers</h3>
+        <ul class="speak-group"></ul>
+        <button type="button" class="more-btn">More ˅ <i id="chevron" class="fas fa-chevron-down" aria-hidden="true"></i></button>
+      </section>
+
+      <!-- partner section -->
+      <section class="partners">
+        <h3 class="sec-title">Partner</h3>
+        <ul class="partner-list">
+          <li><img src="./images/partners/airbnb.svg" alt="airbnb" /></li>
+          <li><img src="./images/partners/google.svg" alt="google" /></li>
+          <li>
+            <img src="./images/partners/att-vector-logo.svg" alt="partners" />
+          </li>
+          <li><img src="./images/partners/mozilla.svg" alt="mozilla" /></li>
+          <li><img src="./images/microsoft-logo.svg" alt="logo" /></li>
+        </ul>
+      </section>
+    </main>
+    <footer>
+        <div>
+            <img src="./images/microsoft-logo.svg" alt="logo" />
+            <p>© Microsoft 2022</p>
+          </div>
+    </footer>
```

Comment:
- [ ] [REQUIRED] Good job so far! As mentioned in the requirement [previous code reviewer](https://github.com/tamoorsaeed22/My-Capstone-Project/pull/1/files#r1169938951) said the create your own `logo` instead of `Microsoft` because logically it is not a clear work, please implement it and change your `Brand` to your own 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/232793216-5c68526b-fa66-4751-bdeb-831e118b80f1.png)

Reply:
@mukhammadrizooff changed the Brand to my own as demanded by both reviewers and personalized my website .